pages/en/developing/supported-networks.mdx

```diff
@@ -8,6 +8,11 @@ The following networks are supported on The Graph's Decentralized Network:
 
 - `mainnet` (Ethereum)
 - `gnosis`\*
+- `celo`\*
+- `celo-alfajores`\*
+- `avalanche`\*
+- `arbitrum-one`\*
+- `arbitrum-goerli`\*
```

Comment:
@MichaelMacaulay – I'm not 100% sure all of these are supported on The Graph Network, do you know?

Reply:
They are supported, @benface. Well, will be as of Monday Feb 27

Reply:
Thank you @cmwhited!